src/lib/crypto/kimchi_bindings/stubs/src/plonk_verifier_index.rs

```diff
@@ -28,13 +29,53 @@ pub enum CamlLookupsUsed {
     Joint,
 }
 
+#[derive(ocaml::IntoValue, ocaml::FromValue, ocaml_gen::Struct)]
+pub struct CamlLookupSelectors<T> {
```

Comment:
I don't think it really matters, but feel free to move it. What would be nicer for maintenance is if everything lived in this repo (or perhaps a separate bindings repo, so that we can test builds in proof-systems CI!) and everything was merged with the wasm code.